CHANGELOG.md

```diff
@@ -44,6 +45,8 @@ using 0.38.0 or lower with blobovniczas configured, please migrate ASAP.
 
 Remove `grpc.tls.use_insecure_crypto` from any storage node configuration.
 
+`neofs-ir` does not use `timers.emit` configuration value anymore.
```

Comment:
i'd add an incentive similar to the previous sentence

Reply:
ok, changed